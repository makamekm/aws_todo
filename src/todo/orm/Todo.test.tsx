import {
    TodoModel,
} from "./Todo";

describe("Todo", () => {
    describe("Orm", () => {

        it("should Todo model match snapshot", async () => {
            const todo = new TodoModel();
            todo.date = new Date("1991/02/16");
            todo.id = 123;
            todo.isDone = true;
            todo.name = "test";
            todo.updated = new Date("1991/02/17");
            expect(todo).toMatchSnapshot();
        });
    });
});
