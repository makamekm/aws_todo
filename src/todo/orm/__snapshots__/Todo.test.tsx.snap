// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Todo Orm should Todo model match snapshot 1`] = `
TodoModel {
  "date": 1991-02-15T23:00:00.000Z,
  "id": 123,
  "isDone": true,
  "name": "test",
  "updated": 1991-02-16T23:00:00.000Z,
}
`;
